packages/tabs/src/index.js

```diff
@@ -6,11 +6,19 @@ const TabsContext = createContext()
 export const Tabs = ({ children, ...props }) => {
   const [active, setActive] = useState()
   const value = React.useMemo(() => ({ active, setActive }), [active])
+  const tabChildren = React.Children.toArray(children).filter(
+    child => child.type.name === 'Tab'
+  )
+  const contentChildren = React.Children.toArray(children).filter(
+    child => child.type.name === 'Content'
+  )
+
   return (
     <TabsContext.Provider value={value}>
       <Box {...props} __css={{ display: 'inline-flex' }}>
-        {children}
+        {tabChildren}
       </Box>
+      {contentChildren}
```

Comment:
Esto se debe wrapear en un componente box como los children, para permitir la customización del componente

Reply:
wrappear los Tabs.Content dentro de un Box también? 
```suggestion
<Box>
      {contentChildren}
</Box>
```